packages/devextreme-scss/scss/widgets/generic/chat/_colors.scss

```diff
@@ -16,6 +16,7 @@ $chat-messagelist-empty-icon-color: null !default;
 $chat-messagelist-empty-icon-background-color: null !default;
 $chat-messagelist-empty-message-color: null !default;
 $chat-messagelist-empty-prompt-color: null !default;
+$chat-messagelist-day-header-color: null !default;
```

Comment:
Don't forget to adjust the color values to support all color schemes. I know that Roman is working on this task. Please extract a separate task to 'check the color of the 'day-header' element in all color schemes.'

Reply:
I believe we should do immediately in context of this task because it's not related to variables Roman's is working on.
Alex, please review on Storybook that color text is expected for all color schemas in all themes

Reply:
>Alex, please review on Storybook that color text is expected for all color schemas in all themes
I'm not sure what color are we expect in different themes. I checked all themes and the text is readable everywhere, is that what u asked to check?

Reply:
We are almost complete with the PR: https://github.com/DevExpress/DevExtreme/pull/28171. It's better to wait until it is merged and then adjust colors fo the generic color schemes and fluent.saas if it needs.